common/services/offenderDetails.js

```diff
@@ -1,16 +1,18 @@
 const { logger } = require('../logging/logger')
 
+const capitalizeName = name => `${name.charAt().toUpperCase()}${name.slice(1).toLowerCase()}`
```

Comment:
Do we want to capitalise here or just do that at the interface layer? (i.e. using Nunjucks built in title filter)

Reply:
I figured if it was single use-case then just here would be ok; more so if we hook a cache up from here too?